website_portal_address/__init__.py

```diff
@@ -1,4 +1,5 @@
 # -*- coding: utf-8 -*-
-# Copyright 2016-Today Serpent Consulting Services PVT. LTD. (http://www.serpentcs.com)
+# Copyright 2016-Today Serpent Consulting Services PVT. LTD.
+# (http://www.serpentcs.com)
```

Comment:
Why editing all these lines to make two lines? IMO this looks dirty & is out of spec for the OCA header.